checklist_plus/text_generation/llm.py

```
@@ -0,0 +1,688 @@
+                        for completion in completion_set:
+                            # Clean the completion
+                            cleaned = completion.strip(' .,!?;:')
+                            full_text = full_text.replace(self.tokenizer.mask_token, cleaned, 1)
+
+                        # Score decreases with position (first suggestion gets highest score)
+                        score = 1.0 - (i * 0.01)  # Smaller decrement for more granular scoring
+                        all_results.append((completion_set, full_text, score))
+                    else:
+                        print(f"Warning: completion set {completion_set} has {len(completion_set)} items but expected {mask_count}")
+
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)